scripts/publish.js

```diff
@@ -195,9 +187,7 @@ if (doPublishCore) {
   console.log('--- Publishing core ---');
   cleanDistribution(distCoreTarget)
     .then(() => buildPackage('core'))
-    .then(() => copyCoreDistributionFiles(coreLibDir, distCoreTarget))
```

Comment:
The files are now published directly from their location in `libs/dist/core`, so no need to copy them anywhere.